### DISCUSSION TITLE
wxMaxima > Maple

### DISCUSSION THREAD

**Discussion by traemand** [wxMaxima-developers/wxmaxima#1673] [Q&A]
Hello,

I'm a student trying to leave behind Maple as my primary CAS, and am instead implementing wxMaxima into my work. 
However, I'm having issues with some basic stuff, and so i thought I'd start this discussion to help myself, and hopefully other future people looking to convert.

First question:
In Maple I would often solve equations the following way:
eq1: 2*x=y
eq2:32/x=y
sol:=solve({eq1,eq2})
assign(sol)

Then x and y would be assigned for future calculations. 

In wxMaxima I'm having trouble doing the same thing.
I have messed around with stuff like lhs(sol[1]):rhs(sol[1]), but without getting the result I want. Also, if I assign the equations with ":", and then define x to a value, I would expect maxima to print eq1 and eq2 with the value of x, but it doesn't print it, and still considers the equation to hold 2 unknown variables. 

What am I doing wrong? 
Do you have a workflow like mine that you can recommend?

**Comment by traemand** [comment]
As a follow up, i've ran into further issues, that are really, really frustrating. 

I'm trying to solve a system of equations, and ive tried both solve() and linsolve(), with no success. I keep getting an empty list in return!

**(%i2)	eqx:5e3+F*sin(theta*%pi/180)-4e3*cos(60*%pi/180)-8e3*cos(30*%pi/180)=0;
(eqx)	F*sin((%pi*theta)/180)-4000.0*sqrt(3)+3000.0=0

(%i3)	eqy:8e3*sin(30*%pi/180)-4e3*sin(60*%pi/180)-F*cos(theta*%pi/180)=0;
(eqy)	-F*cos((%pi*theta)/180)-2000.0*sqrt(3)+4000.0=0

(%i8)	solve([eqx,eqy],[F,theta]);
rat: replaced 3000.0 by 3000/1 = 3000.0

rat: replaced -4000.0 by -4000/1 = -4000.0

rat: replaced 4000.0 by 4000/1 = 4000.0

rat: replaced -2000.0 by -2000/1 = -2000.0
(%o8)	[]**

Solving the system in Maple gives the following set of solutions by simply using solve({eqx,eqy}):
{F=3964.6 , theta=82.2} , {F=-3964.6 , theta=-97.8}

What can i do to get the same output in wxMaxima?

**Comment by gunterkoenigsmann** [comment]
For this kind of problem I would like to forward you to the maxima-discuss making list on https:// maxima.sourceforge.net: wxMaxima is only the graphical user 8minterface whilst Maxima does the actual maths.
